trailblazer/store/crud/read.py

```diff
@@ -4,20 +4,26 @@
 from sqlalchemy import desc
 from sqlalchemy.orm import Query
 
-from trailblazer.constants import TrailblazerStatus
+from trailblazer.constants import JobType, TrailblazerStatus
 from trailblazer.store.base import BaseHandler
 from trailblazer.store.filters.analyses_filters import (
     AnalysisFilter,
     apply_analysis_filter,
 )
-from trailblazer.store.filters.job_filters import JobFilter, apply_job_filter
+from trailblazer.store.filters.job_filters import JobFilter, apply_job_filters
 from trailblazer.store.filters.user_filters import UserFilter, apply_user_filter
 from trailblazer.store.models import Analysis, Job, User
 
 
 class ReadHandler(BaseHandler):
     """Class for reading items in the database."""
 
+    def get_failed_jobs(self, since_when: datetime = None) -> list[dict[str, str | int]]:
+        """Return the number of failed jobs per category (name)."""
+        return self.get_nr_jobs_with_status_per_category(
+            status=TrailblazerStatus.FAILED, since_when=since_when
+        )
```

Comment:
I get confused by this method - why do we have a dict which goes from string to either string or int? Why not str -> str? Also, would prefer if it was called "get_number_of_failed_jobs" or "count_failed_jobs" or something similar. Here I would expect to get a "list[Job]" response.

Reply:
Yes, the type hint is not helpful and the function name is confusing. The method was just moved around in this PR. I'll rename it